views/layout.html

```diff
@@ -0,0 +1,124 @@
+{#- We can't mount GOV.UK Frontend's assets at root as it's done automatically by the application. -#}
+{%- set assetPath = '/govuk/assets' -%}
```

Comment:
do we need the assets? our previous 2 apps didn't have it, unless this is necessary for 3.6.0. I would try deleting it and see if it runs ok.

Reply:
Will try.

Reply:
Done.